mesa/mesaPROTON_OC.py

```diff
@@ -700,6 +700,10 @@ def setup(self, n_agent):
         self.generate_households()
         self.setup_siblings()
         self.setup_employers_jobs()
+        for agent in [a for a in self.schedule.agents if
+                      a.my_job == None and a.my_school == None and a.age() >= 16 and a.age() < self.retirement_age
```

Comment:
Qui dovremmo fare uno shuffle, se ricordo bene (veniva fatto automaticamente solo nell'esecuzione dello step, non ad ogni chiamata)

Reply:
Corretto, questa questione (issue #188) è stata gia risolta in una branch successiva, risolvo anche qui per completezza.